common/src/test/java/com/sequenceiq/cloudbreak/util/DatabaseCommonTest.java

```diff
@@ -0,0 +1,48 @@
+package com.sequenceiq.cloudbreak.util;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+public class DatabaseCommonTest {
+    private static final String ORACLE_URL = "jdbc:oracle:@test.eu-west-1.rds.amazonaws.com:1521/hivedb";
+
+    private static final String MARIADB_URL = "jdbc:mariadb://test.eu-west-1.rds.amazonaws.com:3306/hivedb";
+
+    private static final String MYSQL_URL = "jdbc:mysql://test.eu-west-1.rds.amazonaws.com:3306/hivedb?useSSL=true&requireSSL=false";
+
+    private static final String MYSQL_URL_WITHOUT_PORT = "jdbc:mysql://test.eu-west-1.rds.amazonaws.com/hivedb?useSSL=true&requireSSL=false";
+
+    private static final String POSTGRES_URL = "jdbc:postgresql://test.eu-west-1.rds.amazonaws.com:5432/hivedb";
+
+    @Test
+    public void testGetHostAndPortAndDatabaseName() {
```

Comment:
Split this into separate tests for MySQL, Oracle, and PostgreSQL. It'll help later.

Also, it looks like poor MariaDB was left out :(